app/views/merchant_items/index.html.erb

```diff
@@ -39,4 +39,15 @@
     </section>
     <% end %>
   </tbody>
-</table>
\ No newline at end of file
+</table>
+
+<h1 style="border-bottom: 1px solid black">Top 5 Most Popular Items:</h1>
```

Comment:
I like that you took the extra second to style throughout this embedded ruby file. This is something I want to get more practice in and this example is great inspiration.